webproject/src/main/java/com/google/sps/firebase/Firebase.java

```diff
@@ -59,4 +59,34 @@ public static String sendGetRequest(String urlString) throws IOException {
 
         return jsonResponse;
     }
+
+    public static String sendPatchRequest(String urlString, String requestBody) throws IOException {
```

Comment:
please use firebaseUtil and dependency injection

Reply:
Removed usage of this file.